frontend/src/queries/nodes/DataTable/dataTableLogic.test.ts

```diff
@@ -51,7 +51,7 @@ describe('dataTableLogic', () => {
         logic.mount()
         const builtDataNodeLogic = dataNodeLogic({ key: testUniqueKey, query: dataTableQuery.source })
         await expectLogic(logic).toMount([builtDataNodeLogic])
-        await expectLogic(logic).toMatchValues({
+        await expectLogic(logic).delay(0).toMatchValues({
```

Comment:
I don't really like that these changes were needed, I'm not sure I understand why it was necessary, but it also doesn't seem terrible

Reply:
I guess that's because in the tests everything happened synchronously before, but now the promise mutex adds an extra async hop.... or something like that. In the real world the async query will always add an async hop, so this is probably just test-specific behaviour 🤷